docs/book/src/concepts/login-modes/interactive.md

```diff
@@ -56,4 +56,4 @@ kubelogin remove-cache-dir
 
 ## References
 
-- https://learn.microsoft.com/en-us/python/api/azure-identity/azure.identity.interactivebrowsercredential?view=azure-python
+- https://learn.microsoft.com/en-us/python/api/azure-identity/azure.identity.interactivebrowsercredential?view=azure-python
```

Comment:
are you using windows editor? can you configure it to conform to linux style end of line please?